### PR TITLE
PRM-1776 Added LSP Paid number to audit

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

File: app/models/auditing/UserHasPenaltyAuditModel.scala [hunk 1]
```diff
@@ -46,6 +46,9 @@ case class UserHasPenaltyAuditModel(
       ""
   }
 
+  private val lspChargesPaid: Seq[LSPDetails] = penaltyDetails.lateSubmissionPenalty.map(_.details.filter(point => point.chargeOutstandingAmount.contains(0)
```
Comment (LewisMitchell) [inline app/models/auditing/UserHasPenaltyAuditModel.scala:49]: Could you use count instead of filter?

Comment (LewisMitchell) [reply app/models/auditing/UserHasPenaltyAuditModel.scala:49]: And then change the val name to prefix amount

File: test/models/auditing/UserHasPenaltyAuditModelSpec.scala [hunk 2]
```diff
@@ -279,13 +279,13 @@ class UserHasPenaltyAuditModelSpec extends SpecBase with LogCapturing {
           expiryReason = None,
           appealInformation = None,
           chargeDueDate = Some(LocalDate.of(2022, 1, 1)),
-          chargeOutstandingAmount = Some(200),
+          chargeOutstandingAmount = Some(0),
```
Comment (LewisMitchell) [inline test/models/auditing/UserHasPenaltyAuditModelSpec.scala:282]: The val name suggests the LSP is unpaid, maybe there could be a new val with a paid LSP?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: test/models/auditing/UserHasPenaltyAuditModelSpec.scala (hunk 2 above)
Comment (AndrewRButcher) [reply test/models/auditing/UserHasPenaltyAuditModelSpec.scala:282]: The test it is used suggests there should be paid LSPs "the user has LSPs that are paid and unpaid" so I'll rename the val

File: app/models/auditing/UserHasPenaltyAuditModel.scala (hunk 1 above)
Comment (AndrewRButcher) [reply app/models/auditing/UserHasPenaltyAuditModel.scala]: Done ✅

File: test/models/auditing/UserHasPenaltyAuditModelSpec.scala (hunk 2 above)
Comment (AndrewRButcher) [reply test/models/auditing/UserHasPenaltyAuditModelSpec.scala:282]: Done ✅

File: app/models/auditing/UserHasPenaltyAuditModel.scala [hunk 3]
```diff
@@ -46,6 +46,9 @@ case class UserHasPenaltyAuditModel(
       ""
   }
 
+  private val ammountOfLspChargesPaid: Int = penaltyDetails.lateSubmissionPenalty.map(_.details.count(point => point.chargeOutstandingAmount.contains(0)
```
Comment (LewisMitchell) [inline app/models/auditing/UserHasPenaltyAuditModel.scala:49]: Just a typo in amount

Comment (LewisMitchell) [inline app/models/auditing/UserHasPenaltyAuditModel.scala:49]: Also do we want to filter out any appealed penalties?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: app/models/auditing/UserHasPenaltyAuditModel.scala (hunk 3 above)
Comment (AndrewRButcher) [reply app/models/auditing/UserHasPenaltyAuditModel.scala]: Fixed ✅

Comment (AndrewRButcher) [reply app/models/auditing/UserHasPenaltyAuditModel.scala]: Fixed ✅

**Pull request merged**